src/v2/Apps/Consign/Routes/MarketingLanding/Components/GetPriceEstimate/ConsignPriceEstimateContext.tsx

```diff
@@ -49,8 +59,9 @@ type Action = {
 }
 
 const initialState: State = {
-  artistInsights: null,
+  artistInsight: undefined,
```

Comment:
Note: `undefined` and `null` are being used appropriately and deliberately here:
- when `undefined`, no value has been set and we show the list the default paintings in the price insights window
- when result comes back `artistInsight` is set to that result, or if no result comes back it's set to `null` and we show zero state